scripts/compile_bundle.sh

```diff
@@ -99,44 +107,39 @@ git checkout ${VER_IPEX}
 git submodule sync
 git submodule update --init --recursive
 
-# Install the basic dependency cmake
-python -m pip install cmake
-
 # Compile individual component
 #  LLVM
 cd ../llvm-project
-git config --global --add safe.directory `pwd`
```

Comment:
When compiling in a docker container with root account, this `git config` command is required. Would you add a UID check around it?
```
if [ ${UID} -eq 0 ]; then
    git config --global --add safe.directory `pwd`
fi
```

Reply:
Thanks for clarifying why this is required. This was failing on me when re-running failed build attempts. I will figure out a robust solution.

Reply:
Removing [safe] section contents in ~/.gitconfig should solve the failure that you encountered.

Reply:
After reviewing git's `safe.directory` option i would recommend to add `git config --global --add 'safe.directory=*'` to the `Dockerfile` that calls this script.

The issue isn't that git runs as root, but mixed directory ownership's caused by the way docker mounts storage. This article explains the issue:

https://www.kenmuse.com/blog/avoiding-dubious-ownership-in-dev-containers/

Triggering this issue inside docker depends on the actual docker setup. It would add a lot of complexity to this script to detect such scenario, since it's not simply "running as root" or "running inside docker".

It **might** also be possible to avoid the issue by switching the user 
https://docs.docker.com/engine/reference/builder/#user

In conclusion, i would recommend leaving these commands out of this script to avoid changing the users git config. It's a docker specific issue and should be solved there.

Reply:
Looks good to me. Let's remove the `git config` from this script then.
Would you add a note about invocation of `git config --global --add 'safe.directory=*'` to [installation guide](https://github.com/intel/intel-extension-for-pytorch/blob/xpu-master/docs/tutorials/installation.md#compile-the-bundle-pytorch-torchvision-torchaudio-intel-extension-for-pytorch-with-script)?
Something like `If the user performs compilation is different than that cloned down the repo, you need to invoke git confit ......`